.github/workflows/test-install.yml

```diff
@@ -0,0 +1,30 @@
+name: Installation
+on:
+  schedule:
+    - cron: "0 0 1 * *"  # First day of the month
+
+jobs:
+  run:
+    name: Installation on (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
+    steps:
+    - uses: actions/checkout@v2
+    - run: git fetch --prune --unshallow --tags
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install pip
+      run: |
+        python -m pip install --upgrade pip
+        pip3 install packaging
+    - name: Install package
+      run: pip install -e .
+    - name: Test module load
+      uses: jannekem/run-python-script-action@v1
+      with:
+        script: import {{cookiecutter.repo_name_slug}}
```

Comment:
Looks like this one didn't make it in cookiecutter

Reply:
Weird, I didn't make any modifications just run cookiecutter following the instructions.

Reply:
Can you fix it? Then we can merge this